ads/vendors/seedtag.js

```diff
@@ -24,7 +24,7 @@ export function seedtag(global, data) {
   /*eslint "local/camelcase": 0*/
   global._seedtag_amp = {
     allowed_data: ['adunitId', 'placement', 'publisherId', 'forceCreative'],
```

Comment:
we keep them for compatibility. Ols integration will continue work with those parameters even if we don't use them